src/components/AddProfileForm.css

```diff
@@ -0,0 +1,61 @@
+.Form-Wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.ProfileForm {
+  color: #cc8696;
+  margin: 1px;
+  align-items: center;
+}
+.FormButtons {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+.Upload {
+  display: flex;
+  flex-direction: column;
+  padding-top: 15px;
+  color: #cc8696;
+
+  margin-top: 20px;
+}
+
+.Submit {
+  height: 30px;
+  width: 45px;
+  border-radius: 10px;
+  text-align: center;
+  transition: 1s;
```

Comment:
To make the transition feel a bit smoother you could add an ease-in-out.
And you might have to define a target property, like "all"

Reply:
thank you!